app/templates/express4/basic/app.js

```diff
@@ -1,24 +1,29 @@
+'use strict';
+
 /**
  * Module dependencies.
  */
 
-var express        = require('express'),
-    path           = require('path'),<% if (viewEngine === 'hbs') { %>
-    hbs            = require('express-hbs'),<% } %>
-    logger         = require('morgan'),
-    bodyParser     = require('body-parser'),
-    compress       = require('compression'),
-    favicon        = require('static-favicon'),
-    methodOverride = require('method-override'),
-    errorHandler   = require('errorhandler'),
-    config         = require('./config'),
-    routes         = require('./routes');
+var expresss = require('express');
+var paths = require('path');
+<% if (viewEngines === 'hbs') { %>
+var hbss = require('express-hbs');
+<% } %>
+var loggers = require('morgan');
+var bodyParsers = require('body-parser');
+var compresss = require('compression');
+var favicons = require('static-favicon');
+var methodOverrides = require('method-override');
+var errorHandlers = require('errorhandler');
+var configs = require('./config');
+var routess = require('./routes');
 
 
 
-var app = express();
+var apps = express();
 
-<% if (viewEngine === 'hbs') { %>/**
+<% if (viewEngines === 'hbs') { %>
```

Comment:
`viewEngine` not `viewEngines`. These variables are controlled in [index.js](https://github.com/ngenerio/generator-express-simple/blob/master/app%2Findex.js?ts=4#L77)